This file is part of MXE.
See index.html for further information.

Only advertise -lz for static builds.

--- a/libzip.pc.in	2012-03-15 10:28:05.000000000 +0100
+++ b/libzip.pc.in	2012-12-18 22:48:27.807773881 +0100
@@ -9,6 +9,7 @@
 Name: libzip
 Description: library for handling zip archives
 Version: @VERSION@
-Libs: -L${libdir} -lzip @LIBS@
+Libs: -L${libdir} -lzip
 Cflags: -I${includedir} -I${libincludedir}
+Requires.private: zlib
 
